src/commands/serve/serve.mts

```diff
@@ -71,7 +71,7 @@ const serve = async (options, command) => {
   // Netlify Build are loaded.
   await getInternalFunctionsDir({ base: site.root, ensureExists: true })
 
-  let settings = /** @type {import('../../utils/types.js').ServerSettings} */ ({})
+  let settings = /** @type {import('../../utils/types.js').ServerSettings} */ {}
```

Comment:
nit: Since you're touching this line, could you use the actual type instead of the JSDoc now that this is TypeScript?

Reply:
done in https://github.com/netlify/cli/pull/6183/commits/9eea77d37cb74b878be984a7b82a89cac0693cd6